feature/loan/build.gradle.kts

```diff
@@ -8,14 +8,28 @@
  * See https://github.com/openMF/mobile-mobile/blob/master/LICENSE.md
  */
 plugins {
-    alias(libs.plugins.mifos.android.feature)
-    alias(libs.plugins.mifos.android.library.compose)
+    alias(libs.plugins.mifos.cmp.feature)
+    alias(libs.plugins.kotlin.serialization)
+    id(libs.plugins.kotlin.parcelize.get().pluginId)
```

Comment:
Adding Pacelize plugin this way is cleaner, shorter and more readable
`alias(libs.plugins.kotlin.parcelize)`